foundation/foundation-mda/src/main/resources/templates/deployment/spark-infrastructure/v2/spark.infrastructure.values.yaml.vm

```diff
@@ -1,5 +1,42 @@
-# This file contains the default values for the aiSSEMBLE Spark History Helm chart.
+# This file contains the default values for the aiSSEMBLE Spark Infrastructure Helm chart.
 
 aissemble-spark-history-chart:
-    eventVolume:
-      enabled: true
\ No newline at end of file
+  eventVolume:
+    enabled: true
+
+aissemble-thrift-server-chart:
+  dependencies:
+    packages:
+      - org.apache.hadoop:hadoop-aws:3.3.4
+
+  deployment:
+    envFromSecret:
+      AWS_ACCESS_KEY_ID:
+        secretName: aws-s3-credentials
+        key: aws_access_key_id
+      AWS_SECRET_ACCESS_KEY:
+        secretName: aws-s3-credentials
+        key: aws_secret_access_key
+
+  sparkConf: |
+    spark.hadoop.fs.s3a.endpoint=http://s3-local:4566
+    spark.hadoop.fs.s3a.access.key=#[[${env:AWS_ACCESS_KEY_ID}]]#
```

Comment:
I: The `#[[ ... ]]#` syntax instructs Velocity not to attempt to parse the inner text.